index.js

```diff
@@ -33,7 +34,7 @@ dumbThinky.prototype = {
 
     this.k.schema.createTableIfNotExists(tableName, function (table) {
       if ('id' in fields && fields['id'].fieldType == 'string') {
-        table.uuid('id')
+        table.uuid('id') // FUTURE: the uuid would need to come client-side for this to work
```

Comment:
so, the language is confusing, but I meant client here, as-in the client to the database (which in the perspective of the webapp, means the server).

Basically, the issue is that rethinkdb had `id` fields on models by default that were auto-created `uuid()`s (basically, those long hex-strings, e.g. for invites, etc).  The nice part about them over auto-increment integers is that they can't easily be guessed.  so if you create invite 1 and then 2 and then 3, someone can guess that the next invite will be 4, and in some cases it's a nice feature for that to be random.

Sqlite, and other dbs don't have a uuid() function (postgres does, but my preference is not to depend on using postgres -- sqlite for dev work is nice to be able to use).  My comment here is just a marker for some future work, but unnecessary, i think, for our particular Spoke project in case a user really wants a uuid.  In that case, the javascript in this project itself (i.e. the 'client') would generate the uuid instead of relying on the backend database to do so.  We'd likely use:
https://www.npmjs.com/package/uuid or similar